usr.bin/paste/paste.c

```diff
@@ -113,12 +111,11 @@ typedef struct _list {
 	char *name;
 } LIST;
 
-STAILQ_HEAD(head, _list);
+STAILQ_HEAD(head, _list) static lh;
```

Comment:
This should appear in the previous commit which added the queue head.

`static` comes before the type name by convention, i.e., this should be `static STAILQ_HEAD ...`.

Reply:
Makes total sense. Just fixed it.